backend/routers/manufacturing.py

```
@@ -0,0 +1,470 @@
+            "product_name": product.product_name if product else None,
+            "product_quantity": float(bom.product_quantity),
+            "is_active": bom.is_active
+        })
+    
+    return result
+
+
+@router.get("/bom/{bom_id}", response_model=dict)
+def get_bom(
+    bom_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)